_posts/2015-01-28-basic-examples.md

```diff
@@ -117,9 +117,172 @@ Cycle.run(main, {
 
 Notice the lines we changed, with `NEW!`. We now map `change` events on the checkbox to the `checked` value of the element (the first `map()`) to VTrees displaying that value. However, we need a [`.startWith()`](http://reactivex.io/documentation/operators/startwith.html) to give a default value to be converted to a VTree. Without this, nothing would be shown! Why? Because our `requests` is reacting to `responses`, but `responses` is reacting to `requests`. If no one triggers the first event, nothing will happen. It is the same effect as meeting a stranger, and not having anything to say. Someone needs to take the initiative to start the conversation. That is what `main()` is doing: kickstarting the interaction, and then letting subsequent actions be mutual reactions between `main()` and the DOM Driver.
 
+<h2 id="Displaying data from HTTP requests">Displaying data from HTTP requests</h2>
+
+One of the most obvious requirements web apps normally have is to fetch some data from the server and display that. How would we build that with Cycle.js?
+
+Suppose we have a backend with a database containing 10 users. We want to have a frontend with one button "get a random user", and to display the user's details, like name and email. This is what we want to achieve:
+
+<a class="jsbin-embed" href="http://jsbin.com/yizisa/embed?output">JS Bin on jsbin.com</a>
+
+Essentially we just need to make a request for the endpoint `/user/:number` whenever the button is clicked. Where would this HTTP request fit in a Cycle.js app?
+
+Recall the dialogue abstraction, mentioned also in the previous "checkbox" example. The app generates VTree Observables as "requests to the DOM". But our apps should also be able to generate different kinds of requests. The most typical type of request is an HTTP *request*. Since these are not anyhow related to the DOM, we need a different driver to handle them.
```

Comment:
"not anyhow related to the DOM" -> "not in any way related to the DOM"

You said you wanted nitpicky :)